src/redux/slices/settingsSlice/index.ts

```diff
@@ -228,6 +229,13 @@ export const unlockApp = createAsyncThunk<
   AsyncThunkWithTypes
 >('settings/unlockApp', async (payload, thunkAPI) => {
   try {
+    if (!Config.TRACE_ID) {
+      const { setGlobalError } = payload
+      const errorMessage = 'Trace Id is not valid'
```

Comment:
If we're displaying this, it should be added to `translations`, under name `global_trace_id_error` for example

Reply:
Done!